code/__DEFINES/subsystems.dm

```diff
@@ -1,3 +1,7 @@
+//Update this whenever the db schema changes
+//make sure you add an update to the schema_version stable in the db changelog
+#define DB_MAJOR_VERSION 4
```

Comment:
IIRC these needed to be in `_compile_options.dm` to function and I would imagine it's still the case.

Reply:
not since code/world was moved to code/game/world